src/common/redis_module/ray_redis_module.cc

```diff
@@ -766,6 +766,13 @@ void TableEntryToFlatbuf(RedisModuleKey *table_key,
                                        fbb.CreateVector(data));
     fbb.Finish(message);
   } break;
+  case REDISMODULE_KEYTYPE_EMPTY: {
```

Comment:
Is this ever happening? Presumably not if the alternative before was a fatal error?

Reply:
Yeah, the places where this function was originally called would check first for a null key. I modified the `RequestNotifications` command to remove that check, though.